isis/src/mro/apps/marci2isis/main.cpp

```diff
@@ -153,9 +154,83 @@ void IsisMain() {
     isisCube.findGroup("Instrument").addKeyword(PvlKeyword("Framelets", framelets[i]));
 
     outputCubes[i]->write(origLabel);
-    delete outputCubes[i];
   }
 
+  QString prodId = outputCubes[0]->label()->findGroup("Archive", Pvl::Traverse)["ProductId"][0]; 
+  prodId = prodId.toUpper();
+  vector<int> frameseq;
+  vector<double> exptime;
+  // Populate with first values
+  Pvl *isisLabelInitial = outputCubes[0]->label();
+  PvlGroup &instInitial = isisLabelInitial->findGroup("Instrument", Pvl::Traverse);
+  double exposure = instInitial["ExposureDuration"][0].toDouble() * 1000.0;
+  frameseq.push_back(0);
```

Comment:
ISIS push frame cameras use 1-based framelet indexing. I would change these to 1-based on ingestion.

Reply:
For normality I agree with Jesse, but to minimize the changes now, I think it would be prudent to leave it alone until we make a concerted effort to get all the Marci code twerked.

Reply:
Keeping current functionality to make marcical easier